.github/workflows/android_builds.yml

```
@@ -71,3 +71,9 @@ jobs:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
           scons -j2 verbose=yes warnings=all werror=yes platform=android target=release tools=no
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: android-template
```

```suggestion
          name: ${{github.job}}
```

Seems to work in the same way. The suggestion applies to other jobs. 🙂

---

Done!